04.bowling/bowling.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+score = ARGV[0]
+scores = score.split(',')
+shots = []
+i = 0
+count = 0
+
+scores.each do |s|
+  shots << if s == 'X'
+    10
+  else
+    s.to_i
+  end
+end
+
+while count < 10
+  # 10フレーム目
+  if shots[i] + shots[i + 2] == 10
+    shots[i, 3].sum
+  else
+    shots[i] + shots[i + 2]
+  end
```

Comment:
```suggestion
  # 10フレーム目
  # if shots[i] + shots[i + 2] == 10
  #   shots[i, 3].sum
  # else
  #   shots[i] + shots[i + 2]
  # end
```

じつはこんな感じで、このブロックを全部消してしまっても
プラクティスで例示されている全てのパターンで正しくスコアを計算できるので
消してしまった方が良いです。

たとえば

```ruby
shots[i] + shots[i + 2]
```

のコードは 配列の要素を足し算するけど、結果を出力したり保存したりしていないので
意味がないコードになっています。

この辺はわかりますか？難しかったらもう少し詳細にコメントします！

Reply:
ありがとうございます。本当ですね..
作成した時は代入する文も用意していたのですが、rubocopでUse the return of the conditional...というエラーが出たときに
解決法を勘違いして削除してしまっていたようです。
しかしここの５行がなくても動作していたことに気付かなかったです。
ありがとうございます。
こちらを削除したプログラムで全てのテストプログラムの動作を確認できたので、修正させていただきます。

Reply:
> 作成した時は代入する文も用意していたのですが、rubocopでUse the return of the conditional...というエラーが出たときに

なるほど〜 rubocop に心を掻き乱されるやつですね 😄